src/libraries/System.Data.Common/ref/System.Data.Common.cs

```diff
@@ -1577,6 +1577,7 @@ public enum SchemaType
     public enum SerializationFormat
     {
         Xml = 0,
+        [System.ObsoleteAttribute("SerializationFormat.Binary is obsolete and should not be used. See https://aka.ms/serializationformat-binary-obsolete for more information.", DiagnosticId = "SYSLIB0038")]
```

Comment:
This is a breaking change: it was already made in the src but never shipped in references.

Reply:
Yeah, we did this recently in #39289 as part of obsoleting (and eventually removing) all uses of BinaryFormatter, for 7.0 (so hasn't shipped yet). It's indeed being treated and documented as a breaking change.